rcongui/src/components/form/custom/PlayerAutocompletion.jsx

```diff
@@ -0,0 +1,87 @@
+import React, {useEffect, useMemo, useRef} from "react";
+import {debounce} from "lodash";
+import {handle_http_errors, postData, showResponse} from "@/utils/fetchUtils";
+import Autocomplete from "@mui/material/Autocomplete";
+import Typography from "@mui/material/Typography";
+import {Avatar, Box} from "@mui/material";
+import TextField from "@mui/material/TextField";
+
+export default function PlayerAutocompletion({state, setState}) {
```

Comment:
just nit-picking: I would use `player` and `setPlayer` instead, as it is more meaningful than `state` in this context :)

Reply:
fixed it 🙂